lib/galaxy/model/__init__.py

```diff
@@ -5176,7 +5182,7 @@ def __init__(self, id=None, user=None, tool_id=None, tag_id=None, user_tname=Non
         self.user_value = None
 
 
-class WorkRequestTagAssociation(ItemTagAssociation):
+class WorkRequestTagAssociation(ItemTagAssociation, RepresentById):
```

Comment:
Another class that can be removed, also introduced in commit d1cd4ab and never used.